internal/speakerlist/speakerlist.go

```diff
@@ -53,6 +61,8 @@ func New(logger log.Logger, nodeName, bindAddr, bindPort, secret, namespace, lab
 
 	if labels == "" || bindAddr == "" {
 		level.Info(logger).Log("op", "startup", "msg", "not starting fast dead node detection (memberlist), need ml-bindaddr / ml-labels config")
+		level.Info(logger).Log("op", "startup", "msg", "when memberlist disabled requires speaker pods to run on all nodes")
```

Comment:
"member list address not provided, disabling speaker discovery. Please note that with this configuration MetalLB assumes all the nodes have running speakers".

I wonder if we should document this somewhere.

Reply:
Added something that should appear in https://github.com/metallb/metallb/tree/main/charts/metallb in the Description